app/util/notifications/hooks/index.ts

```diff
@@ -1,53 +1,67 @@
-import { useEffect } from 'react';
-import notifee, { EventType, AndroidImportance } from '@notifee/react-native';
-import Device from '../../../util/device';
-import { STORAGE_IDS } from '../../../util/notifications/settings/storage/constants';
+import { useCallback, useEffect } from 'react';
+import notifee, {
+  Event as NotifeeEvent,
+  EventType,
+} from '@notifee/react-native';
 import NotificationManager from '../../../core/NotificationManager';
 import Routes from '../../../constants/navigation/Routes';
+import { setupAndroidChannel } from '../setupAndroidChannels';
+import { SimpleNotification } from '../types';
 
 const useNotificationHandler = (
-  bootstrapInitialNotification: () => Promise<void>,
+  bootstrapAndroidInitialNotification: () => Promise<void>,
```

Comment:
Is there a reason why the new name is preferred? The hook as such does not appear Android-specific, still.
```suggestion
  bootstrapInitialNotification: () => Promise<void>,
```

Reply:
fixed.